aurelia-slickgrid/src/aurelia-slickgrid/slick-pagination.ts

```diff
@@ -85,11 +85,11 @@ export class SlickPaginationCustomElement {
   }
 
   dispose() {
-    if (this._filterSubcription) {
-      this._filterSubcription.dispose();
+    if (this._filterSubscriber) {
```

Comment:
do you need the null/undefined `if` logic here since you are setting these in the constructor and this will always pass as `true`?

Reply:
I actually didn't test the Build Library, but in some cases I think it might be undefined

Reply:
hopefully it will build or that is an odd typescript error